.gitlab-ci.yml

```
@@ -120,16 +120,16 @@ specs:
     AWS_SECRET_ACCESS_KEY: test
     CAPYBARA_WAIT_TIME_SECONDS: 5
     COVERAGE: 'true'
-    DOCKER_DB_HOST: postgres
+    DOCKER_DB_HOST: db-postgres
     POSTGRES_DB: identity_idp_test
     POSTGRES_USER: postgres_user
     POSTGRES_PASSWORD: postgres_password
     POSTGRES_HOST_AUTH_METHOD: trust
     RAILS_ENV: test
   services:
-    - name: postgres:13.9
+    - name: postgis/postgis:14-3.3
```

It should not be a big deal, but JUST in case there is a backward compatibility issue, we should use PG13 in CI since that is what AuroraDB instances are currently using.

```suggestion
    - name: postgis/postgis:13-3.3
```

---

I believe when we tried that in [16d45dc](https://github.com/18F/identity-idp/pull/8536/commits/16d45dc91e4ae6165464268f345bf5a96a7c1e08) CI failed, but I think my rebase erased that history 😬 I'll try committing this again and let CI run to make sure!

---

13.3 worked after all! 🎉 

The previous failure was likely due to the docker host name issue we fixed later.